### DISCUSSION TITLE
Why does Next add inline styling to <Image /> when rendered?

### DISCUSSION THREAD

**Discussion by costelinivictor** [vercel/next.js#22770] [Help]
The Image Component renders two divs with the img tag inside them. I'm wondering why these divs have inline styling, considering that's not performatic since it cannot be cached by the browser.

**Comment by ErnestBrandi** [comment]
Hi, not sure about it but if it really prevents images to be cached by the browser, this has maybe been done on purpose in order to favor the use of server cache (https://nextjs.org/docs/basic-features/image-optimization#caching).

**Reply by stigkj** [reply]
@ErnestBrandi I don't think this is about caching the image but caching the styling used.
